sycl/test/regression/fsycl-host-compiler-win.cpp

```diff
@@ -25,11 +25,13 @@ int main() {
 
   {
     buffer<int, 1> b(data, range<1>(3), {property::buffer::use_host_ptr()});
+    auto B = b.get_access<access::mode::write>();
     queue q;
     q.submit([&](handler &cgh) {
-      auto B = b.get_access<access::mode::write>(cgh);
-      cgh.parallel_for<class test>(range<1>(3), [=](id<1> idx) {
-        B[idx] = 1;
+      cgh.host_task([=]() {
```

Comment:
Does 'host_task' remains supported even though 'host' is being removed?

Reply:
Yes! host_task does not have anything to do with the host device. However, these tests are arguably not independent from the presence of a device, so they should be moved to the test-suite.

Reply:
`host_task` is executed on `host` and `host` != `host device`.

`host device` term only exists in SYCL 1.2.1 specification and it means the ability to execute SYCL Kernel Functions even if no OpenCL devices are available.  `host` term still exists in SYCL 2020 and `host_task` is related to it, not to `host device`